Toggl.Foundation.MvvmCross/ViewModels/NoWorkspaceViewModel.cs

```diff
@@ -20,28 +24,40 @@ public sealed class NoWorkspaceViewModel : MvxViewModel
         private readonly IAccessRestrictionStorage accessRestrictionStorage;
         private readonly IInteractorFactory interactorFactory;
         private readonly IMvxNavigationService navigationService;
+        private readonly ISchedulerProvider schedulerProvider;
+
         private readonly Subject<bool> isLoading = new Subject<bool>();
 
-        public IObservable<bool> IsLoading => isLoading.AsObservable();
+        public IObservable<bool> IsLoading { get; }
+
+        public UIAction CreateWorkspaceWithDefaultName { get; }
+
+        public UIAction TryAgain { get; }
 
         public NoWorkspaceViewModel(
             ITogglDataSource dataSource,
             IInteractorFactory interactorFactory,
             IMvxNavigationService navigationService,
-            IAccessRestrictionStorage accessRestrictionStorage)
+            IAccessRestrictionStorage accessRestrictionStorage,
+            ISchedulerProvider schedulerProvider)
         {
             Ensure.Argument.IsNotNull(dataSource, nameof(dataSource));
             Ensure.Argument.IsNotNull(accessRestrictionStorage, nameof(accessRestrictionStorage));
             Ensure.Argument.IsNotNull(interactorFactory, nameof(interactorFactory));
             Ensure.Argument.IsNotNull(navigationService, nameof(navigationService));
+            Ensure.Argument.IsNotNull(schedulerProvider, nameof(schedulerProvider));
 
             this.dataSource = dataSource;
             this.accessRestrictionStorage = accessRestrictionStorage;
             this.navigationService = navigationService;
             this.interactorFactory = interactorFactory;
+
+            CreateWorkspaceWithDefaultName = UIAction.FromAsync(createWorkspaceWithDefaultName);
+            TryAgain = UIAction.FromAsync(tryAgain);
+            IsLoading = isLoading.AsDriver(false, schedulerProvider);
```

Comment:
Add an explicit name to the `true` literal please!
```suggestion
            IsLoading = isLoading.AsDriver(onErrorJustReturn: false, schedulerProvider: schedulerProvider);
```